charts/mlp/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
```

Comment:
Are we using still going to use helm 2 for the charts in this repository?

Reply:
updated, it was from the older chart.